packages/boba/account-abstraction/deploy/2-deploy-deposit-paymaster.ts

```diff
@@ -0,0 +1,56 @@
+import { DeployFunction, DeploymentSubmission } from 'hardhat-deploy/types'
+import { Contract, ContractFactory, constants } from 'ethers'
+import { registerBobaAddress } from './1-deploy_entrypoint'
+import BobaDepositPaymasterJson from '../artifacts/contracts/samples/BobaDepositPaymaster.sol/BobaDepositPaymaster.json'
+import MockFeedRegistry from '../artifacts/contracts/test/mocks/MockFeedRegistry.sol/MockFeedRegistry.json'
+
+let Factory__BobaDepositPaymaster: ContractFactory
+let BobaDepositPaymaster: Contract
+
+const deployFn: DeployFunction = async (hre) => {
+  Factory__BobaDepositPaymaster = new ContractFactory(
+    BobaDepositPaymasterJson.abi,
+    BobaDepositPaymasterJson.bytecode,
+    (hre as any).deployConfig.deployer_l2
+  )
+  const entryPoint = await hre.deployments.getOrNull('EntryPoint')
+  console.log(`EntryPoint is located at: ${entryPoint.address}`)
+  let ethPriceOracle = await (hre as any).deployConfig.addressManager.getAddress('FeedRegistry')
+  if (ethPriceOracle === constants.AddressZero) {
```

Comment:
I was thinking if its better to pass the oracle address as an optional env as well? since this is a mock

https://github.com/bobanetwork/boba/blob/aa_devnet/packages/account-abstraction/bin/deploy.ts

Reply:
Added